client/js/components/ReviewScore/ReviewScore.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import styles from './styles';
+
+export default ReviewScore = ({score}) => {
```

Comment:
Make sure to rebase properly. This file has been changed in the Develop branch to remove bugs.

Reply:
Understood.

Reply:
change `export default ReviewScore = ({score}) => {` to `const ReviewScore = ({score}) => {`

and at the end of your file

`export default ReviewScore`